scrapers/mt/bills.py

```diff
@@ -203,8 +228,24 @@ def parse_bill_status_page(self, url, page, session):
             classification = "concurrent resolution"
         elif "r" in _bill_id:
             classification = "resolution"
-
-        chamber = "lower" if _bill_id[0] == "h" else "upper"
+        elif "lc" in _bill_id:
+            classification = "proposed bill"
+
+        # chamber
+        if _bill_id[0] == "h":
+            chamber = "lower"
+        elif _bill_id[0] == "s":
+            chamber = "upper"
+        else:
+            # fall back to using the sponsor's chamber
+            # used for proposed bills aka unintroducd aka LC bills
+            if " HD " in list_sponsor:
+                chamber = "lower"
+            if " SD " in list_sponsor:
+                chamber = "upper"
+            else:
```

Comment:
ah in reviewing the code itself so I could suggest an alternative, I think I missed part of your point on the issue itself.  If I read this right, you're using the sponsor to determine bill chamber, and these can't be determined cleanly?

Reply:
Yes, if an organization is the sponsor, and there does not appear to be a chamber assignment, should I instead default to "legislature" ?

Reply:
yes, let's go with legislature if the chamber isn't known.  sponsor would be the committee 👍

Reply:
Cool, pushed changes - please take a peak and make sure I got it

Reply:
LGTM